packages/hoprd/src/api/v3/paths/messages/index.ts

```diff
@@ -32,7 +32,6 @@ const DELETE: Operation = [
 const POST: Operation = [
   async (req, res, _next) => {
     console.log('SEND 1')
-    const tag = req.body.tag
```

Comment:
This looks like an invalid merge.

Reply:
Tags are not used in the rust based libp2p. I added a log for a tag, but there's no real relevant mechanism to get this through. Is that an important data piece?

Reply:
Yes `tag` is the new feature that is passed down to `ApplicationData` and the Message Inbox uses it to categorize.
Or am I missing something ?

Reply:
Does it have a translatable type or is it a string?

Reply:
I have added a note and this will be sorted once the application is running by introducing a `TaggedMessage`.